Python/getargs.c

```diff
@@ -2831,6 +2831,24 @@ _PyArg_NoPositional(const char *funcname, PyObject *args)
     return 0;
 }
 
+int
+_PyArg_NoTupleKeywords(const char *funcname, PyObject *kwnames)
```

Comment:
@vstinner 
But this is essentially same implemenation of `_PyArg_NoStackKeywords`which is removed.